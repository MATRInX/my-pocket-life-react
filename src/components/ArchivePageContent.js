import React from 'react';
import Archive from './Archive';
import PostList from './PostsList';

const ArchivePageContent = ({ match }) => (
  <div>
    ArchivePageContent content...
    <PostList />
    <Archive selectValue={match.params.month +'|'+match.params.year}/>
  </div>
);

export default ArchivePageContent;