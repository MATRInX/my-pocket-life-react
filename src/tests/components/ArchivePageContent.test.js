import React from 'react';
import { shallow } from 'enzyme';
import ArchivePageContent from '../../components/ArchivePageContent';

test('should correctly render ArchivePageContent', () => {
  const match = {
    params: {
      month: 8,
      year: 2011
    }
  };
  const wrapper = shallow(<ArchivePageContent match={match}/>);
  expect(wrapper).toMatchSnapshot();
});