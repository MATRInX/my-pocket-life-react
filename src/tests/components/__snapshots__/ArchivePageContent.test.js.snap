// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render ArchivePageContent 1`] = `
<div>
  ArchivePageContent content...
  <Connect(PostList) />
  <Connect(Archive)
    selectValue="8|2011"
  />
</div>
`;
